retrospect/20250213.md

```
@@ -0,0 +1,37 @@
+
+### Keep:계속 유지
+  - 공용 함수 사용이 어색할 수 있지만, 공용 함수를 사용하여 코드의 중복과 운영 및 디버깅 난이도를 줄이자.
+
+---
+
+### Problem: 문제가 발생한 행동
+  - 시간이 부족하다는 생각이 들고 있는데, 마일스톤을 빠르게 설정하자.
+
+---
+### Try: 다음 번에 새롭게 시도했으면 좋을 행동
+  - 
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)